src/main/user-impl/java/com/mysql/cj/jdbc/ha/ca/ClusterAwareConnectionProxy.java

```diff
@@ -982,7 +981,6 @@ private void processFailoverFailure(String message) throws SQLException {
     }
 
     this.log.logError(message);
-    releasePluginManager();
```

Comment:
Why we don't need this call?

Reply:
Causes NPE when trying to failover writer (and fails) then trying to execute again (For `conn.close()`) on a null plugin manager.